diff --git a/app/openapi/lib/src/api/default_api.dart b/app/openapi/lib/src/api/default_api.dart
index 3dd9a6d..5d2e828 100644
--- a/app/openapi/lib/src/api/default_api.dart
+++ b/app/openapi/lib/src/api/default_api.dart
@@ -1,9 +1,10 @@
 //
 // AUTO-GENERATED FILE, DO NOT MODIFY!
 //
-
+import 'package:sse_stream/sse_stream.dart';
 import 'dart:async';
-
+import 'dart:convert';
+import 'dart:io';
 import 'package:built_value/json_object.dart';
 import 'package:built_value/serializer.dart';
 import 'package:dio/dio.dart';
@@ -1568,7 +1569,7 @@ class DefaultApi {
   ///
   /// Returns a [Future] containing a [Response] with a [JsonObject] as data
   /// Throws [DioException] if API call or serialization fails
-  Future<Response<JsonObject>> m4tTtsStreamPost({
+  Future<Response<T>> m4tTtsStreamPost<T>({
     M4tTtsStreamPostRequest? m4tTtsStreamPostRequest,
     CancelToken? cancelToken,
     Map<String, dynamic>? headers,
@@ -1595,6 +1596,7 @@ class DefaultApi {
       },
       contentType: 'application/json',
       validateStatus: validateStatus,
+      responseType: ResponseType.stream,
     );
 
     dynamic _bodyData;
@@ -1616,7 +1618,7 @@ class DefaultApi {
       );
     }
 
-    final _response = await _dio.request<Object>(
+    return await _dio.request<T>(
       _path,
       data: _bodyData,
       options: _options,
@@ -1624,37 +1626,6 @@ class DefaultApi {
       onSendProgress: onSendProgress,
       onReceiveProgress: onReceiveProgress,
     );
-
-    JsonObject? _responseData;
-
-    try {
-      final rawResponse = _response.data;
-      _responseData = rawResponse == null
-          ? null
-          : _serializers.deserialize(
-              rawResponse,
-              specifiedType: const FullType(JsonObject),
-            ) as JsonObject;
-    } catch (error, stackTrace) {
-      throw DioException(
-        requestOptions: _response.requestOptions,
-        response: _response,
-        type: DioExceptionType.unknown,
-        error: error,
-        stackTrace: stackTrace,
-      );
-    }
-
-    return Response<JsonObject>(
-      data: _responseData,
-      headers: _response.headers,
-      isRedirect: _response.isRedirect,
-      requestOptions: _response.requestOptions,
-      redirects: _response.redirects,
-      statusCode: _response.statusCode,
-      statusMessage: _response.statusMessage,
-      extra: _response.extra,
-    );
   }
 
   /// 书籍详情
@@ -2800,7 +2771,7 @@ class DefaultApi {
   ///
   /// Returns a [Future] containing a [Response] with a [BuiltList<EbookDto>] as data
   /// Throws [DioException] if API call or serialization fails
-  Future<Response<BuiltList<EbookDto>>> syncFullPost({
+  Stream<BuiltList<EbookDto>> syncFullPost({
     FullSyncDto? fullSyncDto,
     CancelToken? cancelToken,
     Map<String, dynamic>? headers,
@@ -2808,7 +2779,7 @@ class DefaultApi {
     ValidateStatus? validateStatus,
     ProgressCallback? onSendProgress,
     ProgressCallback? onReceiveProgress,
-  }) async {
+  }) async* {
     final _path = r'/sync/full';
     final _options = Options(
       method: r'POST',
@@ -2827,6 +2798,7 @@ class DefaultApi {
       },
       contentType: 'application/json',
       validateStatus: validateStatus,
+      responseType: ResponseType.stream,
     );
 
     dynamic _bodyData;
@@ -2846,7 +2818,7 @@ class DefaultApi {
       );
     }
 
-    final _response = await _dio.request<Object>(
+    final _response = await _dio.request<ResponseBody>(
       _path,
       data: _bodyData,
       options: _options,
@@ -2855,36 +2827,24 @@ class DefaultApi {
       onReceiveProgress: onReceiveProgress,
     );
 
-    BuiltList<EbookDto>? _responseData;
-
-    try {
-      final rawResponse = _response.data;
-      _responseData = rawResponse == null
-          ? null
-          : _serializers.deserialize(
-              rawResponse,
-              specifiedType: const FullType(BuiltList, [FullType(EbookDto)]),
-            ) as BuiltList<EbookDto>;
-    } catch (error, stackTrace) {
-      throw DioException(
-        requestOptions: _response.requestOptions,
-        response: _response,
-        type: DioExceptionType.unknown,
-        error: error,
-        stackTrace: stackTrace,
-      );
+    if (_response.statusCode != 200) {
+      throw Exception('Failed to connect to event stream: Status code ${_response.statusCode}');
     }
 
-    return Response<BuiltList<EbookDto>>(
-      data: _responseData,
-      headers: _response.headers,
-      isRedirect: _response.isRedirect,
-      requestOptions: _response.requestOptions,
-      redirects: _response.redirects,
-      statusCode: _response.statusCode,
-      statusMessage: _response.statusMessage,
-      extra: _response.extra,
-    );
+    await for (final event
+        in _response.data!.stream.cast<List<int>>().transform(Utf8Decoder()).transform(const SseEventTransformer())) {
+      try {
+        if (event.data != null) {
+          final current = _serializers.deserialize(
+            jsonDecode(event.data!),
+            specifiedType: const FullType(BuiltList, [FullType(EbookDto)]),
+          ) as BuiltList<EbookDto>;
+          yield current;
+        }
+      } catch (e) {
+        throw Exception('Failed to parse event stream:  ${event.data} ${e.toString()}');
+      }
+    }
   }
 
   /// 获取系统信息
