diff --git a/app/openapi/lib/src/api/default_api.dart b/app/openapi/lib/src/api/default_api.dart
index 8a5a349..0ad0a26 100644
--- a/app/openapi/lib/src/api/default_api.dart
+++ b/app/openapi/lib/src/api/default_api.dart
@@ -1445,7 +1445,7 @@ class DefaultApi {
   ///
   /// Returns a [Future] containing a [Response] with a [JsonObject] as data
   /// Throws [DioException] if API call or serialization fails
-  Future<Response<JsonObject>> m4tTtsStreamPost({ 
+  Future<Response<T>> m4tTtsStreamPost<T>({ 
     M4tTtsStreamPostRequest? m4tTtsStreamPostRequest,
     CancelToken? cancelToken,
     Map<String, dynamic>? headers,
@@ -1472,6 +1472,7 @@ class DefaultApi {
       },
       contentType: 'application/json',
       validateStatus: validateStatus,
+      responseType: ResponseType.stream,
     );
 
     dynamic _bodyData;
@@ -1492,7 +1493,7 @@ class DefaultApi {
       );
     }
 
-    final _response = await _dio.request<Object>(
+    return await _dio.request<Object>(
       _path,
       data: _bodyData,
       options: _options,
@@ -1501,35 +1502,6 @@ class DefaultApi {
       onReceiveProgress: onReceiveProgress,
     );
 
-    JsonObject? _responseData;
-
-    try {
-      final rawResponse = _response.data;
-      _responseData = rawResponse == null ? null : _serializers.deserialize(
-        rawResponse,
-        specifiedType: const FullType(JsonObject),
-      ) as JsonObject;
-
-    } catch (error, stackTrace) {
-      throw DioException(
-        requestOptions: _response.requestOptions,
-        response: _response,
-        type: DioExceptionType.unknown,
-        error: error,
-        stackTrace: stackTrace,
-      );
-    }
-
-    return Response<JsonObject>(
-      data: _responseData,
-      headers: _response.headers,
-      isRedirect: _response.isRedirect,
-      requestOptions: _response.requestOptions,
-      redirects: _response.redirects,
-      statusCode: _response.statusCode,
-      statusMessage: _response.statusMessage,
-      extra: _response.extra,
-    );
   }
 
   /// 书籍详情
